README.md

```diff
@@ -1,7 +1,7 @@
 # ![Apex](assets/logo.svg)
 
 > **Warning**: This project is currently a PoC. It's an experiment to try out
-> some new ideas. Do not rely on this as it could be abandoned at any time.
+> some new ideas. Do not rely on this, as it could be abandoned at any time.
 
 [![build](https://github.com/redhat-et/apex/actions/workflows/build.yml/badge.svg)](https://github.com/redhat-et/apex/actions/workflows/build.yml)
 
```

Comment:
While you're here, removing the quote on line 8 would be appreciated since it only applied when we had a different name 😆

Reply:
done!